doctr/datasets/datasets/base.py

```diff
@@ -49,6 +51,17 @@ def __getitem__(
             # typing issue cf. https://github.com/python/mypy/issues/5485
             img = self.sample_transforms(img)  # type: ignore[call-arg]
 
+        if isinstance(target, dict) and 'boxes' in target.keys() and \
+                not np.all((target['boxes'] <= 1) & (target['boxes'] >= 0)):
```

Comment:
It would be better to have a dynamic flag: 
```python
if self.convert_to_relative:
```
that would be set in the constructor